spec/factory_specs/legal_aid_application_factory_spec.rb

```diff
@@ -47,4 +47,73 @@
       end
     end
   end
+
+  describe ':with scope limitations' do
+    context "don't specify proceeding types" do
+      context "don't specify the number of proceeding types" do
+        it 'creates one proceeding type' do
+          application = create :application, :with_scope_limitations
+          expect(ProceedingType.count).to eq 1
```

Comment:
With FactoryBot, I don't think you can be absolutely sure that `ProceedingType.count` returns zero at the start of the spec. So this would be safer:
```ruby
let(:application) {  create :application, :with_scope_limitations }
it 'creates one proceeding type' do
  expect { application }.to change { ProceedingType.count }.by(1)
  expect(application.proceeding_types.count).to eq(1)
end
```